tools/wt_to_mdb_bson.py

```diff
@@ -26,7 +26,8 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-import bson, codecs, pprint, subprocess, sys, re
+import argparse, bson, codecs, pprint, subprocess, sys, re
+from bson.json_util import dumps, CANONICAL_JSON_OPTIONS
 from enum import Enum
 
 # This script is intended to parse the output of three wt util commands and convert MongoDB bson
```

Comment:
Could we add something to this comment section to describe the new functionality you've implemented, and some example usage commands?